hooks/boot/06-postgresql-upgrade-extensions.rb

```diff
@@ -12,6 +14,11 @@ def os_needs_postgresql_upgrade?
   def postgresql_upgrade(new_version)
     logger.notice("Performing upgrade of PostgreSQL to #{new_version}")
 
+    start_services(['postgresql'])
```

Comment:
Argh. This won't work in all scenarios. Especially not in those where f-m already did `services stop` and `swtich-to`. Fuck.

Reply:
https://github.com/theforeman/foreman-installer/pull/974